source/administration/changelog.md

```diff
@@ -14,6 +14,7 @@ Mattermost v5.20.0 contains low to high level security fixes. [Upgrading](http:/
 
 ### Breaking Changes
  - Any [pre-packaged plugin](https://docs.mattermost.com/administration/plugins.html#pre-packaged-plugins) that is not enabled in the ``config.json`` will no longer install automatically, but can continue to be installed via the [plugin marketplace](https://docs.mattermost.com/administration/plugins.html#plugin-marketplace).
+ - An unavoidable breaking change was introduced where bool fields on interactive dialogs are now passed as Booleans rather than Strings.
```

Comment:
```
 - Boolean elements from interactive dialogs are no longer serialized as strings. While we try to avoid breaking changes, this change was necessary to allow both the web and mobile apps to work with the boolean elements introduced with v5.16.
```